packages/wonder-blocks-clickable/src/components/__tests__/clickable-behavior.test.js

```diff
@@ -763,7 +778,7 @@ describe("ClickableBehavior", () => {
         expect(onClick).toHaveBeenCalledTimes(expectedNumberTimesCalled);
     });
 
-    it("calls onClick on mouseup when the mouse was dragging", () => {
+    it("does not call onClick on mouseup when the mouse was dragging", () => {
```

Comment:
**suggestion (non-blocking):** I think this test case should be split into three and the titles should be more specific since there's really three different test cases being tested here.

Reply:
Now that I removed `onDrag` altogether, I don't think this can be tested using `fireEvent.dragStart`. Also the middle test case (dragging within the button) doesn't show the same result as manual testing does.

Reply:
I guess I can keep 2/3 of these tests.

Reply:
I added back the two tests that did seem to work. But dragging within the button and letting go should result in a click, which I can't seem to replicate with RTL/jest. I also initially wanted to add a test for pressing in a button, leaving, reentering, and then letting go - this should click, but it doesn't seem to be able to in a jest/RTL test. It works manually though.